code/modules/core_implant/core_implant.dm

```diff
@@ -127,7 +127,7 @@
 			if(R.ignore_stuttering)
 				ture_message = message_pre_problems
 			if(R.compare(ture_message))
-				if(R.power > src.power)
+				if(R.power > src.power && !(power >= max_power))
```

Comment:
This will stop church abilities from being used if someone's at max power. Change the `&&` to an `||` and remove the `!`.